src/xenia/base/debugging_posix.cc

```diff
@@ -9,21 +9,51 @@
 
 #include "xenia/base/debugging.h"
 
-#include <signal.h>
+#include <csignal>
 #include <cstdarg>
+#include <fstream>
+#include <iostream>
+#include <mutex>
+#include <sstream>
 
 #include "xenia/base/string_buffer.h"
 
 namespace xe {
 namespace debugging {
 
-bool IsDebuggerAttached() { return false; }
-void Break() { raise(SIGTRAP); }
+bool IsDebuggerAttached() {
+  std::ifstream proc_status_stream("/proc/self/status");
+  if (proc_status_stream.is_open()) {
```

Comment:
I think you meant `if (!proc_status_stream.is_open()) {` instead

Reply:
Done